docs/src/api/class-response.md

```diff
@@ -43,6 +43,26 @@ Headers with multiple entries, such as `Set-Cookie`, appear in the array multipl
 An array with all the request HTTP headers associated with this response. Unlike [`method: Response.allHeaders`], header names are not lower-cased.
 Headers with multiple entries, such as `Set-Cookie`, appear in the array multiple times.
 
+## async method: Response.headerValue
+- returns: <[null]|[string]>
+
+Returns the first value of the header matching the name. The name is case insensitive. If multiple headers have the same name, for example `set-cookie`, the they are returned as a list separated by `, `. If no header is found, `null` is returned.
```

Comment:
"Returns the first value of the header matching the name." seems wrong.

Reply:
done